e2e/cluster_proxy_test.go

```diff
@@ -55,6 +55,9 @@ func (p *proxyEtcdProcess) Config() *etcdServerProcessConfig { return p.etcdProc
 func (p *proxyEtcdProcess) EndpointsV2() []string { return p.proxyV2.endpoints() }
 func (p *proxyEtcdProcess) EndpointsV3() []string { return p.proxyV3.endpoints() }
 
+// TODO: proxy doesn't provide health information
+func (p *proxyEtcdProcess) EndpointsMetrics() []string { return []string{p.proxyV3.murl} }
```

Comment:
can this panic instead? the metrics tests are !cluster_proxy so it shouldn't hit this path